robotbase/src/main/java/com/team2357/lib/arduino/ArduinoSensor.java

```diff
@@ -0,0 +1,39 @@
+package com.team2357.lib.arduino;
```

Comment:
This should also have been done in it's own branch, as it is not relevant to the shuffleboard.

Reply:
Generally, we will only end up with one copy of this class, and will use it to fulfill functional interfaces for subsystems using a sensor.